transform/mattermost-analytics/models/intermediate/product/boards_active_users/_int_boards_active_users__models.yml

```diff
@@ -0,0 +1,107 @@
+        description: The server's unique id.
+      - name: daily_active_users
+        description: |
+          Total number of active users for the current date and server.
+      - name: weekly_active_users
+        description: |
+          Total number of active users for the current date and past 6 days (7 days total) and for the given server.
+      - name: monthly_active_users
+        description: |
+          Total number of active users for the current date and past 29 days (30 days total) and for the given server.
+
+  - name: int_boards_server_active_days
+    description: |
+      User activity per date and server. Reported by server-side telemetry. This table contains rows only for the 
+      dates/servers that servers reported telemetry on. In case of multiple telemetry events reported by the same server
+      and date, the last in the day is used.
+    columns:
+      - name: server_id
+        description: The server's unique id.
+      - name: server_date
+        description: The date that the user was active on.
+      - name: daily_server_id
+        description: A unique id for each server and date.
+      - name: daily_active_users
+        description: |
+          Total number of active users for the past 24 hours prior to the timestamp of the event.
+      - name: weekly_active_users
+        description: |
+          Total number of active users for the past 30 days prior to the timestamp of the event.
+      - name: monthly_active_users
+        description: |
+          Total number of active users for the past 30 days prior to the timestamp of the event.
+      - name: count_registered_users
+        description: Total number of users, excluding deleted users. Reported by mattermost server.
+
+
+  - name: int_boards_active_days_spined
+    description: |
+      Contains record per user per day since the user's first active date. Uses data from boards only.
+
+    columns:
+      - name: activity_date
+        description: The date that the user was active on.
+      - name: server_id
+        description: The server's unique id.
+      - name: daily_active_users
+        description: |
+          The number of unique board active users for the given server and date. Calculated from user telemetry.
+      - name: weekly_active_users
+        description: |
+          The number of unique board active users for the date and previous 6 days. Calculated from user telemetry.
+      - name: monthly_active_users
+        description: |
+          The number of unique board active users for the date and previous 29 days. Calculated from user telemetry.
+      - name: server_daily_active_users
+        description: |
+          Total number of active users for the past 24 hours prior to the timestamp of the event.
+          Reported by mattermost server.
+      - name: server_weekly_active_users
+        description: |
+          Total number of active users for the past 30 days prior to the timestamp of the event.
```

Comment:
past 7 days?

Reply:
Copy-paste error. Fixed here and in `_product__board__models.yml`.